docs/static/filebeat_modules/mysql/slowlog/pipeline.conf

```diff
@@ -0,0 +1,33 @@
+input {
```

Comment:
This error is expected because https://github.com/elastic/beats/blob/master/filebeat/module/mysql/slowlog/test/mysql-darwin-brew-5.7.10.log#L1-L3 is actually not the logs we are expecting and hence there is no match. The actual slowlog line starts with 
```
# Time: 161209 14:37:59
# User@Host: debian-sys-maint[debian-sys-maint] @ localhost []
```